### ISSUE TITLE
Add ability to disconnect from warp client-side

### ISSUE THREAD

**Issue by DannyBen** [spolu/warp#25]
When a client user connects with `warp connect <id>`, what do they need to do to disconnect?

The usual suspects of `Ctrl+D`, `Ctrl+C` or colon (:) did not work.

The solution, whatever it is, should probably be mentioned in the README.

**Comment by spolu** [comment]
Hi @DannyBen!

Anything that quits the newly opened shell also disconnects the warp, as a warp is a shared shell instance. Do `exit` or `Ctrl+D` should work depending on the shell you use.

**Comment by spolu** [comment]
I'll keep this one opened to update the README

**Comment by DannyBen** [comment]
The connecting client has no way of exiting then?... When sharing a read only warp session, the viewer cannot type `exit` and `Ctrl+D` does nothing.,,, what am I missing? 

We were simply testing it with bash, although now that I think about it, maybe I started the session from within tmux, maybe this caused the other side to not be able to `Ctrl+D`...

**Comment by spolu** [comment]
Fair point. Indeed Ctrl+D does not work client-side and should probably.
It is hinted here[0] that you should simply close your window, but that's definitely not clear.

[0] https://github.com/spolu/warp#automatic-client-terminal-resize

**Comment by spolu** [comment]
Thanks @DannyBen
